docs/developing/add-new-type.markdown

```diff
@@ -2,168 +2,180 @@
+	// in the documentation
+	markers: [],
+	// The tag field is used to contain tag data, this is an array of strings
+	tags: [],
+	// In Jellyfish, contracts are "soft deleted" and the active field is used to
+	// indicate whether or not the contract is in this state or not. An active value of
+	// "false" would indicate that the contract has been deleted
+	active: true,
+	// ??
+	requires: [],
+	// ??
+	capabilities: [],
+	// The data field contains the type definition JSON schema and additional meta
+	// data
+	data: {
+	  // The schema field is a JSON schema defining the shape of a type contract. Type
+	  // contracts are also contracts (just like this one) and are merged against the
+	  // default contract definition for purposes of validation, so you don't need
+	  // to duplicate anything that already exists at the top level.
+	  // This schema is used for validating new Pokémon contracts, and for creating
+	  // the form used to create them.
+	  schema: {
+			type: 'object',
+			required: ['data'],
+			properties: {
+				data: {
+					type: 'object',
+					required: ['firstSeen'],
+					properties: {
+						height: {
+							type: "number"
+						},
+						weight: {
+							type: "number"
+						},
+						description: {
+							type: "string"
+						},
+						abilities: {
+							type: "string"
+						},
+						pokedexNumber: {
+							title: "National Pokèdex Number",
+							type: "number"
+						},
+						caught: {
+							type: "boolean"
+						},
+						firstSeen: {
+							title: "First seen",
+							description: "The first time you saw this pokèmon",
+							type: "string",
+							format: "date-time"
+						}
+					},
+				},
+			},
+		},
+	},
+};
 ```
 
 For the sake of keeping things organised, the name of the file should match the
-"slug" value of the card. In the example shown above, the file would be called
-`pokemon.json`.
+"slug" value of the contract. In the example shown above, the file would be called
+`pokemon.ts`.
 
-## Type cards are loaded when the server starts
+## Type contracts are loaded when the server starts
 
-When the server starts it loads all cards from all plugins. You just need to make sure that your new card is returned by the plugin's `getCards` method. Typically this is done by adding a line to the `lib/cards/index.js` file in the plugin repo.
+When the server starts it loads all contracts from all plugins. You just need to make sure that your new contract is returned as part of the plugin's `contracts` array. Typically this is done by adding a line to the `lib/contracts/index.ts` file in the plugin repo.
 
-Continuing with the Pokèmon card type example, the line would look like this:
+Continuing with the Pokèmon contract type example, the line would look like this:
 
-```javascript
-require('./contrib/pokemon.json'),
+```typescript
+import { pokemon } from './pokemon';
 ```
-or, if the card type made use of mixins:
+<!-- 
+or, if the contract type made use of mixins:
```

Comment:
I think we can drop this mention of mixins. Mixins are now used within the contract definition (`pokemon.ts` in this case) and not when the contracts are imported.

Reply:
Good, just removed that comment